janusgraph-backend-testutils/src/main/java/org/janusgraph/olap/OLAPTest.java

```diff
@@ -139,6 +141,37 @@ private int generateRandomGraph(int numV) {
         return numE;
     }
 
+    @Test
+    public void scannerShouldSeeAllVertices() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5097167)